Source/Libraries/openXDA.Configuration/EmailSection.cs

```diff
@@ -94,6 +94,13 @@ public string Password
         [DefaultValue(false)]
         public bool EnableSSL { get; set; }
 
+        /// <summary>
+        /// Sets the minimum number of samples in an email chart
+        /// </summary>
+        [Setting]
+        [DefaultValue(-1)]
+        public int MinimumChartSamples { get; set; }
```

Comment:
In openSEE, we used a minimum sample rate because there's generally a point where the waveforms look "smooth enough".

This is more like what I've done for downsampling in the past. I used pixel count to determine the level to which data should be downsampled because the chart can't show any more resolution than that anyway. From that perspective, I suppose this works okay if we assume the charts in the email are likely always going to be the same size. If we want to avoid that assumption, we should use something like minimum samples per hundred pixels or even just assume one sample per pixel.

However, I don't think there's any real need to force upsampling to match the number of pixels in the chart the way I did for downsampling. I think it would be best to use `MinSamplesPerCycle`, like in openSEE.

Reply:
Please see newest commit, it should implement what you suggested.